### ISSUE TITLE
Ansible 2.9 bug with php-versions 'Define PHP variables' task

### ISSUE THREAD

**Issue by geerlingguy** [geerlingguy/drupal-vm#1985]
### Issue Type

 - Bug Report / Support Request

### Your Environment

N/A

### Your OS

 - Any

### Full console output

```
TASK [geerlingguy.php-versions : Define PHP variables.] ************************
934ok: [localhost] => (item={'value': u'__php_extension_conf_paths', 'key': u'php_extension_conf_paths'})
935ok: [localhost] => (item={'value': u'__php_fpm_daemon', 'key': u'php_fpm_daemon'})
936ok: [localhost] => (item={'value': u'__php_uploadprogress_module_path', 'key': u'php_uploadprogress_module_path'})
937ok: [localhost] => (item={'value': u'__php_xhprof_module_path', 'key': u'php_xhprof_module_path'})
938fatal: [localhost]: FAILED! => {"msg": "The task includes an option with an undefined variable. The error was: 'php_fpm_conf_path' is undefined\n\nThe error appears to be in '/var/www/drupalvm/provisioning/roles/geerlingguy.php-versions/tasks/main.yml': line 8, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- name: Define PHP variables.\n  ^ here\n"}
```

### Summary

On Debian/Ubuntu base boxes, the above error results after updating to the latest version of the `geerlingguy.php-versions` role, if you're running the latest stable version of Ansible (2.9.0).

This seems like it _could_ be related to an upstream Ansible bug/issue, https://github.com/ansible/ansible/issues/64169

Basically, for some reason, the `php_fpm_conf_path` variable is undefined in the `geerlingguy.php-versions` role.

**Comment by geerlingguy** [comment]
Just pushed a possible fix; needed to use the role-defined variable in the definition of the other role-defined variable. Although I'm a little worried that some variable scoping issues might crop up in Ansible 2.9.x :/

**Comment by geerlingguy** [comment]
That bug is fixed, now hitting a Varnish role error with a missing filter in Ansible 2.9: https://github.com/geerlingguy/ansible-role-varnish/issues/90
